app/views/shared/_flash_banner.html.erb

```diff
@@ -0,0 +1,16 @@
+<% flash.each do |type, msg| %>
+  <% if type == 'alert' %>
```

Comment:
Wondering if we should make this more clear that is a success? the markup is very coupled to a success banner, all green, etc. Alert sounds a bit like not so successful 🤣 
But is just an observation, no blocking.

Reply:
Yeah I was thinking something similar